docs/getting-started/ios/fastlane-swift.md

```diff
@@ -0,0 +1,39 @@
+# Getting Started with Fastlane.swift
+
+Welcome to Fastlane.swift. Fastlane.swift allows you to write your _fastlane_ configuration in Swift, using Xcode and the language you have come to know and love from the world of iOS development.
```

Comment:
"configuration using Xcode, in Swift - the language you have come to know and love from the world of iOS development."

Reply:
We should add a note below this paragraph to make it clear that this feature is in beta.